### PR TITLE
the practical killing apperatuses (pKas, for short) are no longer semi-automatic

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
## About The Pull Request

fun fact: mining 1 tile would give you the mining skill and nerf your ability to use the KA (from 0 to 1.3). same is true for ebows.

ebows no longer scale with mining either, it doesn't make sense.

## Why It's Good For The Game

bug fix and oversight (?) fix.

## Changelog
:cl: Cobby
fix: pKas / Ebows are no longer semi-automatic
del: Ebows no longer scale with mining xp (well if it worked)
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
don't punish us responsible PKA owners for the actions of criminals!

File: code/datums/skills/mining.dm [hunk 1]
```diff
@@ -5,7 +5,7 @@
 /datum/skill/mining/get_skill_speed_modifier(level)
 	switch(level)
 		if(SKILL_LEVEL_NONE)
-			return 1.3
```
Comment (PKPenguin321) [inline code/datums/skills/mining.dm]: wtf is this magic number in the first place

**Comment by Whoneedspacee** [comment]
wow more miner nerfs smh

File: code/datums/skills/mining.dm (hunk 1 above)
Comment (Qustinnus) [reply code/datums/skills/mining.dm]: That's now how modifiers work noob

**Comment by Qustinnus** [comment]
This is a poopier version of nemvars pr

**Comment by nemvar** [comment]
This doesn't fix the issue. The issue is that get skill_modifier returns 0.

**Pull request closed without merging**